contributing.md

```diff
@@ -6,18 +6,20 @@ project you agree to abide by its terms.
 
 ---
 
-Ensure your pull request adheres to the following guidelines:
+To contribute to this repo:
```

Comment:
I'd also describe the option to use the web ui to make changed and commit. This is more accessible I think to less tech savvy people